frontend/src/pages/modelCatalog/ModelCatalogRoutes.tsx

```diff
@@ -1,14 +1,14 @@
 import * as React from 'react';
 import { Navigate, Route, Routes } from 'react-router-dom';
 import ModelCatalogCoreLoader from './ModelCatalogCoreLoader';
-import ModelDetailsPage from './ModelDetailsPage';
+import ModelDetailsPage from './screens/ModelDetailsPage';
 import ModelCatalog from './screens/ModelCatalog';
 
 const ModelCatalogRoutes: React.FC = () => (
   <Routes>
     <Route path={'/:modelCatalogSource?/*'} element={<ModelCatalogCoreLoader />}>
       <Route index element={<ModelCatalog />} />
-      <Route path="tempDetails" element={<ModelDetailsPage />} />
+      <Route path=":sourceName/:repositoryName/:modelName/:tag" element={<ModelDetailsPage />} />
```

Comment:
If we are handling `:sourceName` here, we should remove `:modelCatalogSource?/*` from the above root path and it should just be `path="/"`. It makes sense to do so I think, we only need that param in the root route if we intend to do something with it in ModelRegistryCoreLoader which we don't need to.